Shared/Samples/Show extruded features/ShowExtrudedFeaturesView.swift

```diff
@@ -0,0 +1,121 @@
+    }()
+    
+    /// The renderer of the feature layer.
+    @State private var renderer: SimpleRenderer = {
+        // Setup the symbols used to display the features (US states) from the table.
+        let lineSymbol = SimpleLineSymbol(style: .solid, color: .blue, width: 1.0)
+        let fillSymbol = SimpleFillSymbol(style: .solid, color: .blue, outline: lineSymbol)
+        return SimpleRenderer(symbol: fillSymbol)
+    }()
+    
+    /// The population statistic selection of the picker.
+    @State private var statisticSelection = Statistic.totalPopulation
+    
+    /// A Boolean value indicating whether to show an error alert.
+    @State private var isShowingAlert = false
+    
+    /// The error shown in the alert.
+    @State private var error: Error? {
+        didSet { isShowingAlert = error != nil }
+    }
+    
+    var body: some View {
+        VStack {
+            SceneView(scene: scene)
+                .task {
+                    do {
+                        // Create service feature table from US census feature service.
+                        let featureTable = ServiceFeatureTable(url: .censusMapStates)
+                        try await featureTable.load()
+                        
+                        // Create feature layer from service feature table.
+                        let featureLayer = FeatureLayer(featureTable: featureTable)
+                        
+                        // Feature layer must be rendered dynamically for extrusion to work.
+                        featureLayer.renderingMode = .dynamic
+                        
+                        // Set the renderer scene properties.
+                        renderer.sceneProperties.extrusionMode = .baseHeight
+                        renderer.sceneProperties.extrusionExpression = Statistic.totalPopulation.extrusionExpression
+                        
+                        // Set the renderer on the layer and add the layer to the scene.
+                        featureLayer.renderer = renderer
+                        scene.addOperationalLayer(featureLayer)
+                    } catch {
+                        // Present error if the feature table fails to load.
+                        self.error = error
+                    }
+                }
+                .alert(isPresented: $isShowingAlert, presentingError: error)
+            
+            // Unit system picker.
+            Picker("", selection: $statisticSelection) {
+                Text("Total Population").tag(Statistic.totalPopulation)
+                Text("Population Density").tag(Statistic.populationDensity)
+            }
+            .pickerStyle(.segmented)
+            .padding()
+            .onChange(of: statisticSelection) { _ in
+                if let statistic = Statistic(rawValue: statisticSelection.rawValue) {
+                    renderer.sceneProperties.extrusionExpression = statistic.extrusionExpression
+                }
+            }
```

Comment:
```suggestion
            .onChange(of: statisticSelection) { newValue in
                featureLayer.renderer?.sceneProperties.extrusionExpression = newValue.extrusionExpression
                }
            }
```
If the suggestions above are applied.